cli/src/commands/org.rs

```diff
@@ -0,0 +1,133 @@
+//! Subcommand `phylum org`.
+
+use clap::ArgMatches;
+use reqwest::StatusCode;
+
+use crate::api::PhylumApi;
+use crate::commands::{CommandResult, ExitCode};
+use crate::config::Config;
+use crate::format::Format;
+use crate::{print_user_failure, print_user_success, print_user_warning};
+
+/// Handle `phylum org` subcommand.
+pub async fn handle_org(api: &PhylumApi, matches: &ArgMatches, config: Config) -> CommandResult {
+    match matches.subcommand() {
+        Some(("list", matches)) => handle_org_list(api, matches).await,
+        Some(("link", matches)) => handle_org_link(api, matches, config).await,
+        Some(("unlink", _)) => handle_org_unlink(config).await,
+        Some(("member", sub_matches)) => {
+            let org = match config.org() {
+                Some(org) => org,
+                None => {
+                    print_user_failure!("Arguments missing target organization");
+                    print_user_failure!("Either add `--org` flag, or update your configuration:");
```

Comment:
```suggestion
                    print_user_failure!("Missing target organization");
                    print_user_failure!("Either add `--org` flag, or update configuration:");
```
The first time I encountered this error message, I didn't quite parse it correctly. Namely, that there is an option to add a missing argument and a different option to update the user configuration to set the org globally. This is just a suggestion for making it a bit more clear and shorter.